unmaven-project.sh

```diff
@@ -29,10 +29,9 @@ BUILDXML=build.xml
 
 download_jars() {
     rm -rf $WEBINFLIB $LIBSRC $TESTLIB
```

Comment:
It is better to arrange in the order of `$WEBINFLIB`, `$TESTLIB`, `$LIBSRC` according to the correction below.

Reply:
Fix in 26eaf29cc1d4292b009180e7dfc984d272bd167c